pylutron_caseta/smartbridge.py

```diff
@@ -564,22 +575,40 @@ def _handle_unsolicited(self, response: Response):
     async def _login(self):
         """Connect and login to the Smart Bridge LEAP server using SSL."""
         try:
-            await self._load_devices()
-            await self._load_buttons()
-            await self._load_lip_devices()
-            await self._load_scenes()
             await self._load_areas()
-            await self._load_occupancy_groups()
-            await self._subscribe_to_occupancy_groups()
-            await self._subscribe_to_button_status()
-
-            for device in self.devices.values():
-                if device.get("zone") is not None:
-                    _LOG.debug("Requesting zone information from %s", device)
-                    response = await self._request(
-                        "ReadRequest", f"/zone/{device['zone']}/status"
-                    )
-                    self._handle_one_zone_status(response)
+
+            # Read /project to determine bridge type
+            project_json = await self._request("ReadRequest", f"/project")
+            project = project_json.Body["Project"]
+
+            if (project["ProductType"] == "Lutron Smart Bridge Project"):
```

Comment:
I'd reverse this and try the `project["ProductType"] == "Lutron RadioRA 3 Project"` first

If it doesn't match that, use the existing code path that way we don't break any ones that we don't use that identifier

Reply:
Ok, makes sense.  I'll have a look this evening.

Reply:
I've adjusted as you suggested.